tests/contextmenu.test.js

```
@@ -0,0 +1,276 @@
+import { expect } from 'chai';
+import * as Blockly from 'blockly';
+import { initContextMenus } from '../ui/contextmenu.js';
+import { setBlockLocked, isBlockLocked } from '../ui/blocklyutil.js';
+import { defineControlBlocks } from '../blocks/control.js';
+import { translate } from '../main/translation.js';
+
+export function runContextMenuTests(flock) {
```

ESLint

[error] 'flock' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)